ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/composetesting/SampleTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.dualscreen.composetesting
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import com.microsoft.device.dualscreen.composetesting.ui.theme.ComposeTestingTheme
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+
+class SampleTest {
+    private val composeTestRule = createAndroidComposeRule<MainActivity>()
+    private val publisherRule = createWindowLayoutInfoPublisherRule()
+
+    @get: Rule
+    val testRule: TestRule
+
+    init {
+        testRule = RuleChain.outerRule(publisherRule).around(composeTestRule)
+        RuleChain.outerRule(composeTestRule)
+    }
+
+    @Test
+    fun app_deviceOpenVertically_showTwoPanes() {
+        composeTestRule.setContent {
+            ComposeTestingTheme {
+                ComposeTestingApp()
+            }
+        }
+        // Simulate vertical foldingFeature
+        publisherRule.simulateVerticalFoldingFeature(composeTestRule)
+
+        // Assert both panes are being shown
+        composeTestRule.onNodeWithText(composeTestRule.getString(R.string.pane1_text)).assertIsDisplayed()
+        composeTestRule.onNodeWithText(composeTestRule.getString(R.string.pane2_text)).assertIsDisplayed()
+    }
```

Comment:
If possible, it would be great to add just two more tests - one that does the exact same checks for a horizontal folding feature, and one that doesn't simulate any folding features (this one should check that pane1_text is displayed but pane2_text does not exist)